sdk/js/src/algorand/__tests__/testHelpers.ts

```diff
@@ -366,7 +366,7 @@ export async function signSendAndConfirmAlgorand(
   const result = await waitForConfirmation(
     algodClient,
     txs[txs.length - 1].tx.txID(),
-    1
```

Comment:
does this need to change?

Reply:
My tests were failing without waiting more than one block.  Not sure why it wasn't set to 4 in the first place.  Seems to be a standard for Algorand.

Reply:
👍